addons/coupon/security/ir.model.access.csv

```diff
@@ -0,0 +1,9 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
probably those access rights should be in module sale_coupon, and not in coupon, this will allow you to remove the dependency of sales_team in module coupon as it is not used in the coupon module itself. (and it is already in the dependencies graph through sale_management in sale_coupon)